src/data/forecast.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Minor point but the indentation is off here, should be 2 spaces as with the rest of the project